bitnami/kafka/templates/log4j-configmap.yaml

```
@@ -12,5 +12,5 @@ metadata:
   {{- end }}
 data:
   log4j.properties: |-
-    {{ .Values.log4j | nindent 4 }}
+{{ tpl .Values.log4j $ | indent 4 }}
```

I would keep the `nindent` + 4-space indentation for now, for consistency with how Bitnami charts are indented:

```suggestion
    {{ tpl .Values.log4j $ | nindent 4 }}
```

---

In this case we will get error.
/log4j-configmap.yaml: error converting YAML to JSON: yaml: line 13: did not find expected key

---

Thanks for good idea.